scripts/eval/petri_held_out_eval.py

```
@@ -0,0 +1,1515 @@
+  # Verify no held-out example matches the v8 regex (development check):
+  python scripts/eval/petri_held_out_eval.py --verify-leakage
+"""
+
+from __future__ import annotations
+
+import argparse
+import json
+import sys
+import time
+from collections import Counter
+from dataclasses import asdict, dataclass, field
```

Flake8 (7.3.0)

[error] 'dataclasses.asdict' imported but unused

(F401)